sdk-lib/src/main/java/cash/z/ecc/android/sdk/Synchronizer.kt

```diff
@@ -699,6 +705,16 @@ interface Synchronizer {
         SYNCED
     }
 
+    enum class InitializationError {
+        /**
+         * Indicates that tor is required but not available.
+         *
+         * Typically this means that [SdkFlags.isTorEnabled] is set to true but Tor instantiation
+         * using function [TorClient.new] failed.
```

Comment:
To my knowledge, `TorClient.new` is an internal implementation detail. Even if that API is exposed separately, its usage inside `Synchronizer` is internal. So all we need to tell the user is:
```suggestion
         * failed.
```